lifecycle-operator/main.go

```diff
@@ -119,6 +123,14 @@ func main() {
 	}
 	provider := metric.NewMeterProvider(metric.WithReader(exporter))
 	meter := provider.Meter("keptn/task")
+
+	keptnLifecycleActive, err := meter.Int64Counter(KeptnLifecycleActiveMetric, api.WithDescription("signals that Keptn Lifecycle Operator is installed correctly and ready"))
```

Comment:
Isn't there a possibility to use a boolean meter? We actually do not need a counter, as we have only 2 variants (ready/not ready). Did you investigate this possibility ?

Reply:
Alright I will investigate this possibility